### PR TITLE
Correctly bubble up errors when an integration test fails in a docker container.

### PR DESCRIPTION
**Pull request by ph**
As a retrospect leaving the invocation of `docker-compose logs` was a
bad idea, first it could hide some error in the integration suite and it
could add unnecessary long logs line in the CI job making debugging even harder.
So instead, I've remove it completely and leave up to the person debugging to enabled it back.

### CODE REVIEW

**Comment by ph** [comment]
Note: need backport to 6.0 6.x

**Comment by ph** [comment]
need backport to 6.x and 6.6

File: libbeat/scripts/Makefile [hunk 1]
```diff
@@ -202,7 +202,7 @@ integration-tests: prepare-tests
 .PHONY: integration-tests-environment
 integration-tests-environment:  ## @testing Runs the integration inside a virtual environment. This can be run on any docker-machine (local, remote)
 integration-tests-environment: prepare-tests build-image
-	${DOCKER_COMPOSE} run beat make integration-tests RACE_DETECTOR=$(RACE_DETECTOR) DOCKER_COMPOSE_PROJECT_NAME=${DOCKER_COMPOSE_PROJECT_NAME} || ${DOCKER_COMPOSE} logs --tail 200
+	${DOCKER_COMPOSE} run beat make integration-tests RACE_DETECTOR=$(RACE_DETECTOR) DOCKER_COMPOSE_PROJECT_NAME=${DOCKER_COMPOSE_PROJECT_NAME}
```
Comment (ruflin) [inline libbeat/scripts/Makefile]: I wonder if we could leave it in commented out and with an additional comment. So someone having issues with compose in the future does not have to research again how and where to add this.

Comment (ph) [reply libbeat/scripts/Makefile]: I can leave it commented out.

**Comment by ph** [comment]
@ruflin I have added a comment with the docker-compose logs.

**Pull request merged**